### ISSUE TITLE
Filter: Expose Creation/Modification Year

### ISSUE THREAD

**Issue by er4z0r** [tfeldmann/organize#63]
Hi Thomas, 

Organize **version**: 1.7.0

first of all: Thank you for creating organize! I really like the simple and readable configuration syntax!
I was trying to organize my expenses folder, which looks like this:

```
├── 2018
├── 2019
├── 2020
├── Eigenbeleg-Vorlage.docx
└── unsorted
```
What I want to do is compress the folders for the previous year(s). I tried with a rule like this:
```
  - folders:
      - '~/orga/expenses'
    filters:
      - created:
          years: 1
    actions:
      - echo: 'Folder is ready to be archived {path}'
```
Unfortunately created and lastmodified do not have a `years` parameter:

```
ERROR: __init__() got an unexpected keyword argument 'years'
Full traceback at: /home/er4z0r/.cache/organize/log/organize.log
```
This struck me as inconsistent given that there are params for the smaller time units.

Could you please add the `years` parameter to lastmodified and created filters

**Comment by tfeldmann** [comment]
(As a workaround you can use `days: 365`)

A working example might look like this:
```yaml
rules:
  - folders: '~/Documents/'
    subfolders: true
    filters:
      - extension: pdf
      - created:
          days: 365
    actions:
      - move: '~/Archive/{created.year}/'

```
This will create separate folders for each year and move your files there.

The reason this was left out was that the `datetime.timedelta` object from the standard lib could only handle weeks as the largest duration parameter in order to prevent problems with leap years. 
But you are right, filtering by year should be included. It will be added in the next release of organize 👍

Another problem is that you're trying to target folders in your rule. This is planned but not implemented right now (see #32). So at the moment, you can only archive single files but not complete folders.

**Comment by tfeldmann** [comment]
I just released version 1.8.0 with the new `years`, `months`, `weeks` and `seconds` parameters
